cryptography/src/appleMain/kotlin/com/wire/kalium/cryptography/MLSClientImpl.kt

```diff
@@ -84,9 +84,10 @@ class MLSClientImpl(
         return toCommitBundle(coreCrypto.joinByExternalCommit(toUByteList(publicGroupState), defaultGroupConfiguration))
     }
 
-    override suspend fun mergePendingGroupFromExternalCommit(groupId: MLSGroupId) {
+    override suspend fun mergePendingGroupFromExternalCommit(groupId: MLSGroupId): List<DecryptedMessageBundle>? {
         val groupIdAsBytes = toUByteList(groupId.decodeBase64Bytes())
         coreCrypto.mergePendingGroupFromExternalCommit(groupIdAsBytes)
+        return null
```

Comment:
why null?

Reply:
This the apple implementation which is stuck at an old CoreCrypto version which doesn't have this new API.